### DISCUSSION TITLE
support sorting on nested O2M values

### DISCUSSION THREAD

**Discussion by wc-matteo** [directus/directus#13239] [Feature Requests]
support for sorting on nested values has been [added](https://github.com/directus/directus/pull/12084) recently. It doesn't allow sorting on nested O2M values, though.

I think it would be a useful feature.

For example:

> A collection which has a translations collection attached. You want to query the main collection, sorting the results by a field in the translations collection (e.g. updated_on) and filtering by language id so that the translations would only contain a single value (making it, logically speaking, a o2o relation)

**Comment by licitdev** [comment]
Support for sorting on nested O2M values has been added in `9.22.0` via https://github.com/directus/directus/pull/14798.
